erpnext/manufacturing/doctype/production_plan/production_plan.json

```diff
@@ -301,13 +302,20 @@
    "label": "Warehouses",
    "options": "Production Plan Material Request Warehouse",
    "read_only": 1
+  },
+  {
+   "depends_on": "eval: doc.get_items_from == \"Sales Order\"",
+   "fieldname": "sales_order_status",
+   "fieldtype": "Select",
+   "label": "Sales Order Status",
+   "options": "\nDraft\nOn Hold\nTo Deliver and Bill\nTo Bill\nTo Deliver\nCompleted\nCancelled\nClosed"
```

Comment:
Remove the following options:
- Draft
- Cancelled

Since we only create Production Plan for `docstatus == 1`, we should not give it as a filter too.

Also remove **Closed**, **Completed**, and **On Hold**